settings_base.py

```diff
@@ -130,3 +130,5 @@
 # During testing, you may want to use Django's built-in static file server.
 SERVE_STATIC = False
 STATIC_BASE = 'static/'
+
+FASTLANE_USER_NAME = 'user'
```

Comment:
I don't really know why we need `base_settings.py` AND `local_template/settings.py`.

I guess the first is to provide [Django](https://github.com/lukpueh/custominstallerbuilder/blob/master/settings_base.py#L20-L51) and [Seattle](https://github.com/lukpueh/custominstallerbuilder/blob/master/settings_base.py#L54-L94) settings that should not (but still can) be overridden as well as [override-able sensible](https://github.com/lukpueh/custominstallerbuilder/blob/master/settings_base.py#L102)  default settings. Whereas the second is for convenience to help the CIB operator decide which settings s/he might want to customize and at the same time keep a copy of `default values`.

IMHO we could replace `local_template/settings.py` with `local/settings.py`. Because, if we want to provide defaults, we provide them in `settings_base.py`; and if we want to provide `default examples`  that the CIB operator *has* to change (e.g. [`SECRET_KEY`](https://github.com/lukpueh/custominstallerbuilder/blob/master/settings_base.py#L108)) , we provide them as comments in  `settings_base.py` or `local/settings.py`.

This would also eliminate [step 4. in the deployment instruction](https://github.com/SeattleTestbed/docs/blob/master/Operating/CustomInstallerBuilder/Installation.md#deploying-and-running-custom-installer-builder).